config/PartitionLayoutLocalOnePartitionThreeReplicasWithSsl.json

```diff
@@ -0,0 +1,31 @@
+{
+    "clusterName": "LocalSetUp_ThreeNodeOneDisk_OnePartitionThreeReplicas",
+    "version": 1,
+    "partitions": [
+        {
+            "id": 0,
+            "partitionState": "READ_WRITE",
+            "replicaCapacityInBytes": 10737418240,
+            "replicas": [
+                {
+                    "hostname": "localhost",
+                    "mountPath": "/tmp/1",
+                    "port": 6670,
+                    "sslport": 7670
```

Comment:
I don't think the partition layout has ssl ports. It is the hardware layout. Look at HardwareLayout.json and HardwareLayoutSsl.json (notice that there is only PartitionLayout.json and no PartitionLayoutSsl.json). @nsivabalan should be able to confirm.